README.md

```diff
@@ -1,5 +1,16 @@
 # REopt API Analysis using Python
 
+## Updated for Multi-Scenario and Multi-Tariff Inputs
+This fork of the repository includes an updated version of `src.multi_site_input_parser.py`. In particular, it allows the addition of a column in the multi-scenario input file titled `rate_file` that allows a specific electric tariff to be specified for each scenario. Below is an example of some inputs:
```

Comment:
Thank you for updating the README! Can you please modify the text to reflect what you would want as a new user of the capability (and remove the mention of the fork)?